src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -133,6 +137,14 @@
                                                            (.toFixed (/ input-amount
                                                                         conversion-rate)
                                                                      crypto-decimals)))
+             locked-greater-then-send-amount?           (let [amount      (money/bignumber
+                                                                           amount-in-crypto)
+                                                              send-amount (money/bignumber
+                                                                           send-amount-in-crypto)]
+                                                          (if (and (money/bignumber? amount)
+                                                                   (money/bignumber? send-amount))
+                                                            (money/greater-than amount send-amount)
+                                                            false))
```

Comment:
I think this `if` can be simplified to just use the `and`:

```clojure
(and (money/bignumber? amount)
     (money/bignumber? send-amount)
     (money/greater-than amount send-amount))
```

Reply:
Great suggestion, thanks!